gems/aws-sdk-core/lib/aws-sdk-core/process_credentials.rb

```diff
@@ -0,0 +1,54 @@
+module Aws
+  class ProcessCredentials
+
+    include CredentialProvider
+    include RefreshingCredentials
+
```

Comment:
Could you also add some doc here like an example of how to use this?
e.g https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/AssumeRoleCredentials.html